runtime/src/accounts_index.rs

```diff
@@ -705,6 +705,19 @@ impl<
         T: 'static + Clone + IsCached + ZeroLamport + std::marker::Sync + std::marker::Send + Debug,
     > AccountsIndex<T>
 {
+    fn allocate_accounts_index(bins: usize) -> (LockMapType<T>, PubkeyBinCalculator16) {
+        let account_maps = (0..bins)
+            .into_iter()
+            .map(|_| RwLock::new(AccountMap::default()))
+            .collect::<Vec<_>>();
+        let bin_calculator = PubkeyBinCalculator16::new(bins);
+        (account_maps, bin_calculator)
+    }
+    pub fn allocate_index(&mut self, bins: usize) {
```

Comment:
Where does `allocate_index()` get used? I'm guessing that this will be the non-test way to allocate the index, and _not_ the Default way above?

Reply:
Got rid of it - it isn't used in the further work built on this anyway. Good catch.